### ISSUE TITLE
WEBPACK_EXTERNAL_MODULE__support_pe_wasm is not defined

### ISSUE THREAD

**Issue by mvrtich** [Esri/arcgis-webpack-plugin#8]
Trying to use the esri/geometry/projection module:

```
import EsriMap from 'esri/Map';
import MapView from 'esri/views/MapView';
import SpatialReference from 'esri/geometry/SpatialReference';
import * as projection  from 'esri/geometry/projection';

const map: EsriMap = new EsriMap({
  basemap: 'topo-vector'
});

const view: MapView = new MapView({
  map,
  container: 'root'
});

const outSpatialReference: SpatialReference = new SpatialReference ({
  wkid: 4326
});

view.when(() => {
  if(projection.isSupported()) {
    projection.load().then(() => {
      view.on('pointer-move', (event) => {
        const outPoint = projection.project(view.toMap({x: event.x, y: event.y}), outSpatialReference);
        console.log(JSON.stringify(outPoint));
      });
    });
  } else {
    console.log('local projection engine is not supported')
  }
});
```

ReferenceError: __WEBPACK_EXTERNAL_MODULE__support_pe_wasm__ is not defined
    at eval (pe-wasm":1)
    at Object../support/pe-wasm (index.bundle.js:9000)
    at __webpack_require__ (index.bundle.js:62)
    at eval (pe.js:25)

Uncaught (in promise) ReferenceError: __WEBPACK_EXTERNAL_MODULE__support_pe_wasm__ is not defined
    at eval (pe-wasm":1)
    at Object../support/pe-wasm (index.bundle.js:9000)
    at __webpack_require__ (index.bundle.js:62)
    at eval (pe.js:25)

**Comment by odoe** [comment]
Fixed in `4.7.2`

Refer to updated doc on [Node Globals](https://github.com/Esri/arcgis-webpack-plugin#node-globals)

```
// webpack.config.js
  node: {
    process: false,
    global: false,
    fs: "empty"
  }
```

Thanks!

**Comment by tgeorges** [comment]
Hi @odoe, 
I have a not so special case where main code is loaded from  a webserver port 80, and ArcGIS API is loaded from this server (for example webpack dev server) on port 8080.
In this case, ArcGIS API try to load pe asm on webserver with port 80 : it seams that asm loader in module node_modules/arcgis-js-api/geometry/support/pe-wasm.js does not use dojo config base url to fetch this module.
Do you confirm it ? Does it make sense for you to fix it in the API ?

Without that we cannot use that feature of the api :-(

Thanks to you
